auth/auth.go

```diff
@@ -1,21 +1,50 @@
 package auth
 
 import (
+	"io/ioutil"
+	"net/http"
+
+	jwt "github.com/dgrijalva/jwt-go"
+	"github.com/golang/glog"
 	"github.com/rancherio/host-api/app/common"
 	"github.com/rancherio/host-api/config"
-	jwt "github.com/dgrijalva/jwt-go"
-	"net/http"
 )
 
+var fileBytes []byte
+
 func Auth(rw http.ResponseWriter, req *http.Request) bool {
```

Comment:
All of this logging needs to be removed as this will all be logged on each request.  It's excessive.